_posts/0000-01-02-NatiDagosti.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text - I am so lost
```

Comment:
Me too, Natalia from 5 seconds to the past, me too. Sincerely me from the future